Framework/PythonInterface/mantid/plots/__init__.py

```diff
@@ -194,23 +194,48 @@ def pcolormesh(self, *args, **kwargs):
         else:
             return Axes.pcolormesh(self, *args, **kwargs)
 
+    def imshow(self, *args, **kwargs):
+        '''
+        If the **mantid** projection is chosen, it can be
+        used the same as :py:meth:`matplotlib.axes.Axes.imshow` for arrays,
+        or it can be used to plot :class:`mantid.api.MatrixWorkspace`
+        or :class:`mantid.api.IMDHistoWorkspace`. You can have something like::
+
+            import matplotlib.pyplot as plt
+            from mantid import plots
+
+            ...
+
+            fig, ax = plt.subplots(subplot_kw={'projection':'mantid'})
+            ax.imshow(workspace) #for workspaces
+            ax.imshow(C)     #for arrays
+            fig.show()
+
+        For keywords related to workspaces, see :func:`mantid.plots.plotfunctions.pcolormesh`
```

Comment:
For keywords related to workspaces, see :func:`mantid.plots.plotfunctions.imshow`